cloudant-client/src/main/java/com/cloudant/client/api/ClientBuilder.java

```diff
@@ -106,6 +111,38 @@
  */
 public class ClientBuilder {
 
+
+    private static final UserAgentInterceptor USER_AGENT_INTERCEPTOR;
+
+    //init the string, based on a properties file or fallback to some defaults
+    static {
```

Comment:
I think it would be nicer to keep this block with the `UserAgentInterceptor`. We could accept a prefix string in the constructor (potentially with another constructor or helper method for loading the properties).
I guess that static method means reading the properties fewer times, but we will only new the interceptor up once per client anyway.
Then we can reuse more code in sync-android.